package.json

```diff
@@ -57,6 +57,7 @@
     "mocha": "^2.3.4",
     "phantomjs": "^1.9.18",
     "po-loader": "^0.3.1",
+    "po2json": "^0.4.2",
```

Comment:
`po2json` is a required peer for the `po-loader` If you don't include this any fresh build will fail.